uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -78,7 +78,7 @@ my $notify_notsummary = 'N'; # notification_spool message flag for messages to b
 
 =head3 new($dbhr, $debug, $TmpDir, $logfile, $verbose) >> (constructor)
 
-Creates a new instance of this class. The parameter `\$dbhr` is a reference
+Creates a new instance of this class. The parameter C<$dbhr> is a reference
```

Comment:
one line later, not sure if it is worth putting DBI in between C<>?

Reply:
done